src/main/java/FerbFileHandler.java

```diff
@@ -0,0 +1,47 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FerbFileHandler {
+    private String filepath;
```

Comment:
Should this part be filePath instead?
```suggestion
    private String filePath;
```